zipline/pipeline/factors/technical.py

```diff
@@ -144,6 +148,205 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class CorrelationFactor(SingleInputMixin, CustomFactor):
```

Comment:
I'm always in favor of limiting scope initially, but if it's a small amount of work Spearman Rank would certainly be something people would use.